modules/cmdb/services/member/member.go

```diff
@@ -728,9 +728,9 @@ func (m *Member) IsAdmin(userID string) bool {
 		}
 	}
 	if member.ID == 0 {
-		logrus.Warnf("CAUTION: user(%s) current no admin, will become soon", userID)
+		logrus.Warnf("CAUTION: user(%s) currently not an admin, will become soon if no one admin exist", userID)
 		// TODO: some risk
-		if m.noOneAdmin() && len(userID) > 11 {
+		if len(userID) > 11 && m.noOneAdminForKratos() { // len > 11 imply that is kratos user
```

Comment:
why 11

Reply:
just a magic value, kratos' user_id is UUID, it is significantly larger than 11

Reply:
please comment it clearly for latter coders.

Reply:
please comment it clearly for latter coders.

Reply:
you're right, added